libs/remix-debug/test/decoder/localsTests/structArray.ts

```diff
@@ -103,7 +103,7 @@ module.exports = function (st, privateKey, contractBytecode, compilationResult,c
                 st.equals(locals['arrayStruct'].value.b.value[0].value, '34')
                 st.equals(locals['arrayStruct'].value.b.value[1].value, '-23')
                 st.equals(locals['arrayStruct'].value.b.value[2].value, '-3')
-                st.equals(locals['arrayStruct'].value.c.value, 'three')
```

Comment:
weren't you able to find an index were this var is set to `three`?

Reply:
this is fixed